frontend/src/components/containers/ContainerHierarchy.js

```diff
@@ -86,8 +86,8 @@ const renderSample = (sample, sampleKind) => {
          }
          {' '}
         <b>{sample.name}</b> sample ({sampleKind}){' '}
-        {sample.coordinates &&
-          `@ ${sample.coordinates}`
+        {sample.coordinate &&
+          `@ ${withCoordinate(coordinatesByID, sample.coordinate, coordinate => coordinate.name, "Loading...")}`
```

Comment:
In the future, please use WithXXXRenderComponent. I'm planning on refactoring all of the old-style `withXXX` calls in components to use the render component.